packages/console/src/containers/AppContent/components/Topbar/TenantSelector/components/TenantEnvTag/index.module.scss

```diff
@@ -0,0 +1,25 @@
+@use '@/scss/underscore' as _;
+
+.tag {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: _.unit(0.25) _.unit(1.5);
+  border-radius: _.unit(1);
+
+  .text {
+    font: var(--font-body-3);
+  }
+
+  &.development {
+    background: #5d34f21f;
+  }
+
+  &.staging {
+    background: #ffb95a2e;
+  }
+
+  &.production {
+    background: #68be6c2e;
+  }
```

Comment:
Should use an existing color token, as the color is likely to be different in dark and light themes